SPEC.md

```diff
@@ -217,40 +243,37 @@ Success is indicated by a zero return code and the following JSON being printed
 ```
 {
   "cniVersion": "0.1.0",
```

Comment:
@steveeJ I since the new fields don't overwrite the old ones, we could make it backwards compatible by specifying the mapping between old and new (eg, first IP address of sandbox interface goes into the old 'ipv4' and 'ipv6' blocks).  But I think in the last meeting we discussed this and decided it wasn't necessarily worth it?  Need clarification.